### DISCUSSION TITLE
css framework

### DISCUSSION THREAD

**Discussion by crewZLD** [community/community#65176] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Is there any other css framework besides bootstrap?
I'm currently learning web programming

**Comment by nirav-gajera** [comment]
[Tailwind CSS](https://tailwindcss.com/): Tailwind CSS is a utility-based framework that allows you to create custom styles with a minimal amount of code. It is becoming increasingly popular due to its flexibility and performance

**Reply by crewZLD** [reply]
thank you

**Comment by arafajs** [comment]
@crewZLD 

- Bootstrap
- Tailwind CSS
- Foundation
- Bulma
- Semantic UI

**Reply by crewZLD** [reply]
thank youu

**Comment by maulanaibnu** [comment, marked as answer]
If you are just learning web programming, I suggest you use tailwind css or materialize.

**Reply by crewZLD** [reply]
thank you
